Libraries/Hotcakes.Commerce.Dnn/Web/HotcakesModuleBase.cs

```diff
@@ -160,6 +160,12 @@ protected void RegisterScript(string path, int order = 0)
                 FileOrder.Js.DefaultPriority + order);
         }
 
+        protected void RegisterExternalScript(string path, int order = 0)
+        {
+            ClientResourceManager.RegisterScript(Page, ResolveUrl(path),
```

Comment:
The other methods that do the same/similar thing don't have a check to see if the `path` is null or empty, but we should begin adding this.  

Please add a check for `string.IsNullOrEmpty(path)` and throw an `ArgumentNull()` exception if it's empty.  Any string value you might use when doing this should be a constant.